src/libs/getPlatform/index.ts

```diff
@@ -1,6 +1,10 @@
+import * as Browser from '@libs/Browser';
 import CONST from '@src/CONST';
 import type Platform from './types';
 
-export default function getPlatform(): Platform {
+export default function getPlatform(treatMWebDifferently = false): Platform {
```

Comment:
Would you mind renaming this param to `shouldMobileWebBeDistinctFromWeb`?

Reply:
Done. And I was thinking for a smaller name because `treatMWebDifferently` was already mouthful.

Reply:
Variable naming is one of the hardest things! Luckily code-minifiers usually render the name length insignificant in the end, so I prefer more clarity. Thanks!